solutions/standard/variables.tf

```diff
@@ -150,7 +150,7 @@ variable "skip_en_kms_auth_policy" {
 # COS
 ########################################################################################################################
 
-variable "existing_cos_instance_crn" {
+variable "cos_instance_id" {
```

Comment:
@Ak-sky why are you changing this? For the DAs we are keeping consistent with naming, so hence we use `existing_<service>_instance_crn`